### DISCUSSION TITLE
Windows系统下通过Docker部署

### DISCUSSION THREAD

**Discussion by renli19670413** [lobehub/lobehub#5200] [Self Hosting] [answered]
我的电脑是Windows11专业工作站版，我希望通过Docker在本地部署lobechat，请问应该如何操作？

**Comment by SpeedupMaster** [comment, marked as answer]
参考这个文档
https://lobehub.com/zh/docs/self-hosting/server-database/docker-compose
使用 `docker-compose.yml` 部署，然后配置casdoor和minio。更新 `.env` 中的casdoor和minio的相关变量，然后重启

**Comment by renli19670413** [comment]
安装应该没问题了，启动进入本地服务初始界面似乎也是正确的，但是登录出错，是否和配置casdoor和minio有关？应该如何配置？




谢谢！






--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Lumia  ***@***.***>
 

 
 发送时间: 2024-12-27 19:47:35
 

 
 收件人:lobehub/lobe-chat ***@***.***> 

 
 抄送:任力 ***@***.***>,Author ***@***.***> 

 
 主题: Re: [lobehub/lobe-chat] Windows系统下通过Docker部署 (Discussion #5200)
 





 参考这个文档
 https://lobehub.com/zh/docs/self-hosting/server-database/docker-compose
 使用 docker-compose.yml 部署，然后配置casdoor和minio。更新 .env 中的casdoor和minio的相关变量，然后重启
